TRITON/iocBoot/iocTRITON-IOC-01/st-common.cmd

```diff
@@ -32,12 +32,12 @@ ReadASCIIConfigure("$(READASCII_NAME)", "$(RAMP_DIR)", 20)
 ##ISIS## Load common DB records 
 < $(IOCSTARTUP)/dbload.cmd
 
-dbLoadRecords("../../db/TRITON.db", "P=$(MYPVPREFIX)$(IOCNAME):,RECSIM=$(RECSIM=0),DISABLE=$(DISABLE=0),PORT=$(DEVICE),IPADDR=$(IPADDR=NUL)")
-dbLoadRecords("../../db/TRITON_channels.db", "P=$(MYPVPREFIX)$(IOCNAME):,PORT=$(DEVICE)")
-dbLoadRecords("../../db/TRITON_pid.db", "P=$(MYPVPREFIX)$(IOCNAME):,PORT=$(DEVICE)")
-dbLoadRecords("../../db/TRITON_pid_lookup.db", "P=$(MYPVPREFIX)$(IOCNAME):,PORT=$(DEVICE),READ=$(READASCII_NAME),RAMPLIST=$(FILELIST_NAME),RAMP_FILE_NAME=$(RAMP_FILE_NAME=Default.txt),USE_RAMP_FILE=$(USE_RAMP_FILE=0)")
-dbLoadRecords("../../db/TRITON_temp_channels.db", "P=$(MYPVPREFIX)$(IOCNAME):,PORT=$(DEVICE)")
-dbLoadRecords("../../db/TRITON_pressure_channels.db", "P=$(MYPVPREFIX)$(IOCNAME):,PORT=$(DEVICE)")
+dbLoadRecords("../../db/TRITON.db", "P=$(MYPVPREFIX)$(IOCNAME):,RECSIM=$(RECSIM=0),DISABLE=$(DISABLE=0),PORT=$(DEVICE),IPADDR=$(IPADDR=NUL),POLL_RATE=$(POLL_RATE=1 second),CHANNEL=POLL_RATE=$(CHANNEL_POLL_RATE=15 second)")
```

Comment:
Just a thought, do we need to specify the default here AND in the db file? I think for debugging it might be better to just specify it once rather than in each record?